nh_clinical/wizard/user_allocation.py

```diff
@@ -493,16 +493,29 @@ def submit_users(self, cr, uid, ids, context=None):
         if not isinstance(ids, list):
             ids = [ids]
         wiz = self.browse(cr, uid, ids[0], context=context)
-        respallocation_pool = self.pool[
+        reallocation_pool = self.pool[
```

Comment:
Why was this renamed? The Responsibility allocation model is not the same as the reallocation model.

Reply:
Looked like a typo. Will rename.